store/view/categories.js

```diff
@@ -13,32 +13,46 @@ const actionNames = {
 }
 
 // Mutation types
-const types = {
+export const types = {
+  CHANGE_CURRENT_CATEGORY_CODE: 'CHANGE_CURRENT_CATEGORY_CODE',
   LOAD_CATEGORIES: 'LOAD_CATEGORIES',
   ADD_CODES: 'ADD_CODES',
 }
 
 // Mutation function
+const changeCurrCategory = storeHelper.createMutationFn(types.CHANGE_CURRENT_CATEGORY_CODE, 'currentCategoryCode')
```

Comment:
`changeCurrCategory` => `changeCurrentCategory` ? :)

Reply:
Thanks I've changed it a little bit :D